index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+</html>
+<body>
+  <header class="header">
+    <nav class="header_nav">
+      <div class="header_logo black"><a class="black" href="#">Welcome</a></div>
+      <div class="header_menu">
+        <img class="menu_icon" src="images/Union.png" alt="menu" />
+      </div>
+    </nav>
```

Comment:
- [ ] Good job @JansueT :+1: , however the link to the hamburger image is broken. Kindly provide a proper url :pray: 

Your design
 
![Screenshot from 2022-12-02 19-31-28](https://user-images.githubusercontent.com/80458199/205352126-dd3f7219-28e8-4e62-abb8-3cff9bb58962.png)